mockgcp/README.md

```diff
@@ -80,26 +82,36 @@ If you also use `E2E_GCP_TARGET=real` you can run against the real (non-mocked)
 Usually however, this is not necessary; the most common failure mode is that terraform or KCC expects a field to be automatically populated,
 and it normally logs an error like "foo not set" (in this case, simply add that to your mock implementation.)
 
-## Capture HTTP golden logs
+## Capture golden object and HTTP golden logs
+
+1. Capture golden object and HTTP golden logs against real GCP.
 
-Capture HTTP logs against real GCP:
+   1. Run the following command to generate the golden object (`_generated_object_\[testname\].golden.yaml` file):
+      `E2E_KUBE_TARGET=envtest RUN_E2E=1 E2E_GCP_TARGET=real GOLDEN_OBJECT_CHECKS=1 WRITE_GOLDEN_OUTPUT=1 go test -test.count=1 -timeout 3600s -v ./tests/e2e -run TestAllInSeries/fixtures/[testname]`.
```

Comment:
Looks good, though you should also check out `hack/record-gcp`.  I think this maps to `hack/record-gcp fixtures/[testname]`